### ISSUE TITLE
Errors without slug column on model table

### ISSUE THREAD

**Issue by pduersteler** [shioyama/friendly_id-mobility#16]
I am just trying to implement mobility together with friendly id for one model that needs translations. The model that I try to adapt did not had either included before, other models already use friendly_id (without mobility) sucessfully, though.

As far as I understood how this all works, is: When using the key/value store of mobility, each attribute of the model that needs translation capabilities is stored in the mobility translation tables. The same is for friendly_id-mobility.

In my case, I am using the default key/value store for mobility, and added the following to my model:

    class Category < ApplicationRecord
      extend Mobility
      translates :name, :slug, dirty: true

      extend FriendlyId
      friendly_id :name, use: :mobility
    end

Assuming that I don't need the slug column on the categories table (because I can drop the name column as well and everything seems to work through mobility), I tried to generate the slugs by calling `.save` on my existing records, which lead to:

    Unknown column 'categories.slug' in 'where clause'

Adding the `slug` column resolves the error and makes everything work as intended, however the slug column on the model is never being used, which lead me to drop it again and try again, and I got my error back. So now I have a column on my model that feels like it is there for a workaround-reason instead of a real reason.

This may not be a bug but a simple misunderstanding, but I could not find any section in the README and thought I'd ask because I'd like to understand (and possibly update the README or add a wiki page) how it works and why it is that way.

So, long story short: Why and how? I'd like to be enlightened.

Thanks!

**Comment by shioyama** [comment]
Hi there! :wave: Thanks for posting the issue.

I will admit that I haven't touched this code for quite a while, but looking at the specs, none of the [models](https://github.com/shioyama/friendly_id-mobility/blob/c2ecd7639f196a9295ff09ff0550d9d7ab759c63/spec/spec_helper.rb#L18-L68) have any slug column on their model tables, so I have a hard time understanding what's happening in your case.

Can you post some test code that reproduces the problem?

**Comment by pduersteler** [comment]
Hi 👋 

I just tried to set up a minimal reproducible example and I can't reproduce the error there. I assume there is something wrong in my configurations so I'll keep on debugging.

Thanks for the gem, by the way!

**Comment by shioyama** [comment]
Thanks for updating! Ok let me know if you do end up finding any issue with the gem. :smile:
